docs/post-call-survey.md

```diff
@@ -1,63 +0,0 @@
-# How to Configure Post-Call Survey
```

Comment:
Woops sorry I meant remove this file from the PR aka undo the changes, but all good since we have the other PR to add this back anyways